brave-spark-framework-filter/src/test/java/com/github/kristofa/brave/spark/ITBraveSparkTest.java

```diff
@@ -0,0 +1,66 @@
+package com.github.kristofa.brave.spark;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.http.ITHttpServer;
+import com.github.kristofa.brave.http.SpanNameProvider;
+import org.junit.After;
+import org.junit.AssumptionViolatedException;
+import org.junit.Test;
+import spark.ExceptionHandler;
+import spark.Request;
+import spark.Response;
+import spark.Spark;
+
+public class ITBraveSparkTest extends ITHttpServer {
+
+    /**
+     * after filter executed before NotFoundRoute,there is no way to add 404 code after serverspan
+     * has been setServerSend,may be throw a not found exceeption and process it in ExceptionHandler

```

Comment:
nit: watch for punctuation and spelling errors like exceeption and lack of space after comma